README.rst

```diff
@@ -1164,6 +1164,17 @@ Choose from the list of available rules:
 
   *Risky rule: this fixer may change functions named setUp or tearDown outside of PHPUnit tests, when a class is wrongly seen as a PHPUnit test.*
 
+* **php_unit_static_assertion_calls**
+
+  PHPUnit assertion calls must all be of the same type.
```

Comment:
what type ?

Reply:
It is specified in the Configuration options

Reply:
you read description to determine do you want to use this rule and maybe configure it, user won't read options definition if he can't understand description

Reply:
Description updated, hope is enough now.